README.md

```diff
@@ -28,6 +28,31 @@ After you merge new changes:
 
 ```npm run cs import``` 
 
+### Data importing and exporting for development
+As part of this project the development team has created a migration file to allow for consistent data for development and testing. This file will be included in any handover for this project, so please look there for the relevant file.
```

Comment:
I think just a line at the end of this one saying: "paste the provided copy into the root of your local repo. This file will be ignored so as to avoid pushing data to github"